docs/tutorials/issue-tracker/06-frontend.md

```diff
@@ -66,35 +66,57 @@ issue.spec.status = newStatus;
 ```
 
 Finally, we have to display the information. We define one more function:
-```TypeScript
+```tsx
 const getActions = (issue: Issue) => {
-    if(issue.spec.status === 'open') {
+    if (issue.spec.status === 'open') {
```

Comment:
This whole section is actually just copied verbatim from [the file you add itself](https://github.com/grafana/grafana-app-sdk/blob/main/docs/tutorials/issue-tracker/frontend-files/main.tsx#L71), though I get the intent of making it easier to read in the doc.

Reply:
👍 , I'll make sure the TS in the source files gets formatted in the same consistent way.